### DISCUSSION TITLE
Tuya Devices

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [custom-components/ble_monitor#1253] [General]
How come there is no tuya devices that have been developed for this integration??

**Comment by Ernst79** [comment]
There has been a request (https://github.com/custom-components/ble_monitor/issues/1063) in the past, but I need some real BLE data to create this. If you can give me some data from a sensor, I can give it a try. 

https://github.com/custom-components/ble_monitor/issues/1063

**Reply by cloudbr34k84** [reply]
I have a Tuya Lock i was hoping to integrate?

**Reply by cloudbr34k84** [reply]
have you seen this project ?
https://github.com/PlusPlus-ua/ha_tuya_ble
